TLM/TLM/Custom/PathFinding/CustomPathFind.cs

```diff
@@ -289,8 +291,16 @@ private struct BufferItem {
                 disableMask_ |= NetSegment.Flags.Flooded;
             }
 
+            allowMixedCargoPath_ = false;
+            mixedPathVehicleTypes_ = queueItem_.vehicleType;
             if ((laneTypes_ & NetInfo.LaneType.Vehicle) != NetInfo.LaneType.None) {
                 laneTypes_ |= NetInfo.LaneType.TransportVehicle;
+
+                // allowed mixed vehicle path when requeted path is Vehicle + CargoVehicle (at current state CargoTruck and PostVanAI)
```

Comment:
requeted -> requested

Reply:
allowed -> allow

Reply:
if you replace `at current state` with `E.g.:` your comment will be future proof :)